chef_master/source/publish_cookbooks_multiple_servers.rst

```diff
@@ -0,0 +1,83 @@
+===============================================================
+Publish Cookbooks to Multiple Chef Servers
+===============================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/publish_cookbooks_multiple_servers.rst>`__
+
+.. tag chef_automate_mark
+
+.. image:: ../../images/chef_automate_full.png
+   :width: 40px
+   :height: 17px
+
+.. end_tag
+
+The ``delivery-sugar`` cookbook exposes some libraries and a resource that you can use to
+publish a cookbook (or multiple cookbooks) to multiple Chef servers or organizations. The following examples show how to publish to both a single Chef server and how to extend that methodology to publish to multiple Chef servers.
+
+Prerequisites 
+===============================================================
+
+Before you begin, you must copy the ``knife.rb`` file 
+and the client key .pem file (referenced as ``client_key`` in ``knife.rb``) to the build nodes/runners that you will use in the build-cookbook. This can 
+be done manually by logging in to the build nodes/runners and copying the files.
```

Comment:
I would probably mention that this process could be automated.. But I guess it is not very important for this topic, at the end of the day users will try to automated so it is up to you to add it or not, I'm fine either way.